author/ietf/ref/document-attributes.adoc

```diff
@@ -223,85 +235,77 @@ RFC XML: `rfc@indexInclude`
+// `:rfcprocack:`::    if there already is an automatically generated Acknowledg(e)ment section, pluralize its title and add a short sentence acknowledging that `xml2rfc` was used in the document's production to process an input XML source file in RFC-2629 format
 `:slides:`::        when producing a html file, produce multiple files for a slide show
-`:sort-refs:`::     (`sortrefs`) sort references
 `:strict:`::        try to enforce the ID-nits conventions and DTD validity
-`:subcompact:`::    if compact is "yes", then you can make things a little less compact by setting this to "no" (the default value is the current value of the compact PI)
-`:sym-refs:`::      (`symrefs`) use anchors rather than numbers for references
-`:text-list-symbols:`::     modify the list of symbols used (when generated text) for list type="symbols". For example, specifying "abcde" will cause "a" to be used for 1st level, "b" for the 2nd level, etc, cycling back to the first character "a" at the 6th level. Specifying "o*" will cause the characters "o" and "*" to be alternated for each successive level.
+`:subcompact:`::    if compact is `yes`, then you can make things a little less compact by setting this to `no` (the default value is the current value of the compact PI)
+`:text-list-symbols:`::     modify the list of symbols used (when generated text) for list `type="symbols"`. For example, specifying `abcde` will cause "`a`" to be used for 1st level, "`b`" for the 2nd level, etc, cycling back to the first character "`a`" at the 6th level. Specifying `o*` will cause the characters "`o`" and `"*"` to be alternated for each successive level.
 `:toc-include:`::   (`toc`) generate a table-of-contents
-`:tocappendix:`::   control whether the word "Appendix" appears in the table-of-content
-`:toc-depth:`::     if toc is "yes", then this determines the depth of the table-of-contents
-`:tocindent:`::     if toc is "yes", then setting this to "yes" will indent subsections in the table-of-contents
+`:tocappendix:`::   control whether the word "`Appendix`" appears in the table-of-content
+//TODO: `:toc-depth:`::     if `:toc-include:` is "yes", then this determines the depth of the table-of-contents
```

Comment:
@opoudjis the description for `:toc-depth:` is missing, but is still referred to in the next paragraph.